tf2onnx/graph.py

```diff
@@ -492,6 +501,11 @@ def make_const(self, name, np_val, skip_conversion=False, raw=True):
             skip_conversion: bool, indicate whether this created node would be mapped during conversion.
             raw: whether to store data at field of raw_data or the specific field according to its dtype
         """
+
+        key = str(np_val) + "_" + str(np_val.dtype)
+        if key in self._consts:
+            return self._consts[key]
+
```

Comment:
It looks like 'name' will be ignored of subsequent const tensors, if their value is reserved already. So, we cannot create  a const with a specific name that we want (if it's value is already in the _consts dict). 

Don't know if that's OK or not -- could be a perfectly valid thing to do, but just not fully sure if it'll break initialiizers.

Reply:
Reverting back this part, good point!